cachi2/core/package_managers/npm.py

```diff
@@ -534,7 +679,9 @@ def fetch_npm_source(request: Request) -> RequestOutput:
     )
 
 
-def _resolve_npm(pkg_path: RootedPath) -> ResolvedNpmPackage:
+def _resolve_npm(
+    pkg_subpath: Path, source_dir: RootedPath, output_dir: RootedPath, npm_deps_dir: RootedPath
```

Comment:
Why does this method need both output_dir and npm_deps_dir? Shouldn't it need only one of them?

Same for source_dir / pkg_subpath, but that's more relevant in the next comment

Reply:
ou sorry output_dir is a leftover and yes pkg_subpath is sufficient. Fixed